### DISCUSSION TITLE
想把这个网页设置成桌面壁纸

### DISCUSSION THREAD

**Discussion by chenqian3788** [dukeluo/wai#98] [General]
如题，但是我Ctrl+S保存到本地后，html文件就不更新时间了。请问朋友们。有没有办法移植到桌面壁纸使用

目前我的办法是：使用第三方壁纸软件LivelyWallpaper，新建一个网页链接的主题来使用。

但是这样自己没办法自定义图片背景和文本位置和颜色。所以想请教如何移植成壁纸

**Comment by dukeluo** [comment]
现在歪脖子还没有自定义的能力，另外我也很好奇网页是如何移植成桌面壁纸的🤔️

**Reply by chenqian3788** [reply]
额，那个壁纸软件内置了一个Chrome内核，然后只需要软件内新建自定义壁纸时，给它一个链接，壁纸软件就会把网页显示成桌面壁纸，而且还是可以点击交互的！

**Comment by chenqian3788** [comment]
对了，希望能调一个黑暗模式
😊
